.github/workflows/_test_pax.yaml

```diff
@@ -73,11 +87,13 @@ jobs:
             test-pax.sh \
               --output /output/${{ steps.meta.outputs.TEST_CASE_NAME }} \
               --dtype bfloat16 \
-              --batch-per-gpu 8 \
+              --batch-per-gpu 4 \
               --steps 500 \
-              --data-parallel ${{ matrix.DP }} \
-              --tensor-parallel ${{ matrix.TP }} \
-              --pipeline-parallel ${{ matrix.PP }} 
+              --data-parallel ${{ matrix.PARALLEL_CONFIG[0] }} \
+              --tensor-parallel ${{ matrix.PARALLEL_CONFIG[1] }} \
+              --pipeline-parallel ${{ matrix.PARALLEL_CONFIG[2] }} \
+              --nodes ${{ steps.meta.outputs.NODES }} \
+              --multiprocess
```

Comment:
I'm not sure for Pax, but for T5X, the behavior is different with and without `--multiprocess` for single-node-multi-GPU runs. How does Pax behave here?

Reply:
Yes, the behavior is different for the two. I think it is desirable to do the single-gpu run without multiprocess. Would you prefer that I split that run into a separate job?

Reply:
We don't have to split the job. Something along the line `$([[ <SOME-CONDITION> ]] && echo --multiprocess)` should do.